model/cicd/spans.yaml

```diff
@@ -0,0 +1,26 @@
+groups:
+  - id: span.cicd.pipeline.run.server
+    type: span
+    span_kind: server
+    brief: 'Semantic Convention for pipeline runs'
+    stability: development
+    attributes:
+      - ref: cicd.pipeline.result
```

Comment:
please also add `error.type` and describe which `cicd.pipeline.result` should be reported as errors

Reply:
Done in 4a803cee2e1de393676d7d2b9e5b9cd650f07d40.